indico/modules/events/contributions/operations.py

```diff
@@ -110,6 +117,7 @@ def update_contribution(contrib, contrib_data, custom_fields_data=None):
                                      'contribution_id': contrib.id,
                                      'session_block_id': current_session_block.id if current_session_block else None,
                                      'force': True}
+    _handle_legacy_description(contrib)
```

Comment:
I don't think this is the right place. It should be done in the WTForm or the controller - otherwise *any* update will change the render mode without an actual conversion. I think it'd be better to add this to the relevant forms:

```
@generated_data
def render_mode(self):
    return RenderMode.markdown
```